.github/workflows/CI.yml

```diff
@@ -56,4 +51,27 @@ jobs:
         if: matrix.version == '1' && matrix.os == 'ubuntu-latest' && matrix.num_threads == 1
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
-          path-to-lcov: lcov.info
\ No newline at end of file
+          path-to-lcov: lcov.info
+
+  docs:
+    name: Documentation
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install dependencies for GR backend.
+        # See https://github.com/JuliaPlots/Plots.jl/issues/1649#issuecomment-682147970 for details;
+        # also, see https://gr-framework.org/julia.html#installation.
+        run: |
+          sudo apt-get install -y libxt6 libxrender1 libxext6 libgl1-mesa-glx libqt5widgets5
```

Comment:
These are heavy dependencies and should not be needed. I've never had to install them in CI. The suggested environment variable (which was also suggested by the developer of GR in the linked issue) should be sufficient.

```suggestion
```

In my experience, plotting also still works, even with the error messages.